pantheon-bundle/frontend/src/app/__snapshots__/searchFilter.test.tsx.snap

```diff
@@ -152,7 +152,7 @@ exports[`SearchFilter tests should render default Search component 1`] = `
       <FormSelectOption
         isDisabled={false}
         key="Module type"
```

Comment:
Can we also consider changing other module instances changes within this PR?

Reply:
changing the key value breaks the sort.